.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
```

Comment:
@rexwangcc should we run our tests in gitlab instead?

Reply:
Good question, to me it depends on:
- Are we still paying for Travis?
- Do we think it's important to expose test logs to external contributors? (Gitlab is not open to the public)
- Do we want to mirror every repo to Gitlab? I'm not sure if we can run unit tests in Gitlab without having a repo there.
- How many concurrent runners available on Gitlab.

Reply:
@rexwangcc I don't think we're paying for Travis but could be wrong. I think gitlab actually allows for readonly access to the builds if configured. This repo will probably already be mirrored there soon for CI/CD. Also, we can work with BrianH to increase # of workers as needed over time

Reply:
All make sense!